content/tutorials/bundle/index.md

```diff
@@ -2,19 +2,21 @@
 title: "Getting started with the Gatling bundle"
 menutitle: "Bundle"
 slug: "getting-started-with-gatling-bundle"
-description: "Learning how to setup FrontLine Cloud using the Gatling bundle"
+description: "Learn how to setup Gatling Enterprise using the Gatling bundle"
 date: 2021-03-19T17:03:31+01:00
 lastmod: 2021-03-19T17:03:31+01:00
 weight: 50
 ---
 
 {{< include introduction.md >}}
 
-There are multiple ways to create a Gatling artifact, which will be used by FrontLine to launch a run. We can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
+## Download the zip bundle
 
-In this tutorial, we'll use the Gatling bundle. Make sure that you have a JDK 8 or 11 installed on your computer.
+There are multiple ways to create a Gatling artifact, which will be used by FrontLine to launch a run. 
+We can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
+
+In this tutorial, we'll use the **Gatling bundle**. Make sure that you have a JDK 8 or 11 installed on your computer.
```

Comment:
Not part of this PR, but we should have a tutorial on how to setup jdk / gatling_home on windows.